tasks/inc_cloud_iface.yml

```diff
@@ -13,6 +13,7 @@
     command: "{{ item.command|default(omit) }}"
     env: "{{ item.env|default(omit) }}"
     links: "{{ item.inks|default(omit) }}"
+    network_mode: "{{ item.network_mode|default(provision_docker_network) }}"
```

Comment:
I would maybe do it in the same manner as other extra params are:
```
network_mode: "{{ item.network_mode|default(omit) }}"
```
instead of adding new role parameter, but it is up to you ... I am fine with that as it is.